### DISCUSSION TITLE
[docs] Windows + LLVM instructions improvements

### DISCUSSION THREAD

**Discussion by merlinND** [mitsuba-renderer/mitsuba3#175] [Feature requests]
- [ ] Clarify the supported VS versions. In at least one instance, VS 2022 worked whereas VS 2019 didn't.
- [ ] Repeat the suggested LLVM installation instructions from the [DrJit docs](https://drjit.readthedocs.io/en/latest/firststeps-py.html#llvm-backend)
- [ ] Clarify how to proceed if LLVM is installed another way: set the `DRJIT_LIBLLVM_PATH` to point to the location of `LLVM-C.dll`.
    - [ ] It seems that installing with `conda` doesn't work: https://github.com/mitsuba-renderer/mitsuba3/issues/153#issuecomment-1199722702
    - [ ] Point out the behavior of environment variables on Windows vs new terminal sessions.
- [ ] Make sure there's only one set of `setpath` files in the actual build folder, not both `build/` and `build/Release`

**Comment by njroussel** [comment]
> Make sure there's only one set of `setpath` files in the actual build folder, not both `build/` and `build/Release`

It's unfortunate, but I don't think there's a way we can get around this. There's some [specific CMake limitations at play](https://github.com/mitsuba-renderer/mitsuba3/blob/master/CMakeLists.txt#L208-L215). I've at least updated the compilation instructions in the documentation with cc089045
